pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -266,7 +266,7 @@ let
       google_api_key = "AIzaSyDGi15Zwl11UNe6Y-5XW_upsfyw31qwZPI";
 
       # Optional features:
-      use_gio = gnomeSupport || chromiumVersionAtLeast "99";
+      use_gio = true;
```

Comment:
(Note: I decided to leave it under "Optional features" for now to avoid rebuilds and will take a look at it later - we can probably drop it now)